sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -357,17 +451,45 @@ def _predict(self, fh, X=None):
             ),
         )
 
-        pred = pred.sequences.mean(dim=1).detach().cpu().numpy().T
+        pred = pred.sequences.mean(dim=1).detach().cpu().numpy()
 
-        pred = pd.Series(
-            pred.reshape((-1,)),
-            index=ForecastingHorizon(range(len(pred)))
+        ins = np.array(
+            list(np.unique(hist_y.index.droplevel(-1)).repeat(pred.shape[1]))
```

Comment:
Is list necessary or would it work also directly to pass it to the np.array?

Reply:
It's necessary, otherwise, it will be something like this:

```py
[("h0_0", "h1_0")",
 ("h0_0", "h1_0")"
 ...
]
```
`ins.shape` will be incorrect.

It should be something like this:
```py
[['h0_0' 'h1_0']
 ['h0_0' 'h1_0']
 ['h0_0' 'h1_0']
 ...
 ['h0_4' 'h1_9']
 ['h0_4' 'h1_9']
 ['h0_4' 'h1_9']]
```